neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import datajoint as dj
+
+
+from neuro_data.static_images.data_schemas import StaticScanCandidate, StaticScan, ConditionTier, Frame, InputResponse, Eye, Treadmill, StaticMultiDataset, StaticMultiDatasetGroupAssignment
+
+
+class NeuroDataPipelineManagement():
+    def __init__(self):
+        self.pipeline_anatomy = dj.create_virtual_module('pipeline_anatomy', 'pipeline_anatomy')
+        self.pipeline_fuse = dj.create_virtual_module('pipeline_fuse', 'pipeline_fuse')
+        self.pipeline_stimulus = dj.create_virtual_module('pipeline_stimulus', 'pipeline_stimulus')
+
+    def manually_insert_area_for_scan(self, target_scan, area):
+        neuron_unit_keys = (self.pipeline_fuse.ScanSet().Unit() & target_scan).fetch('KEY')
+        for neuron_unit_key in neuron_unit_keys:
+            neuron_unit_key['brain_area'] = area
+            self.pipeline_anatomy.AreaMembership().insert1(neuron_unit_key, allow_direct_insert=True)
+
+    def manually_insert_layer_for_scan(self, target_scan, layer):
+        neuron_unit_keys = (self.pipeline_fuse.ScanSet().Unit() & target_scan).fetch('KEY')
+        for neuron_unit_key in neuron_unit_keys:
+            neuron_unit_key['layer'] = layer
+            self.pipeline_anatomy.LayerMembership().insert1(neuron_unit_key, allow_direct_insert=True)
+
+    # populate functions
+    def process_static_scan(self, target_scan):
+        print('Running preprocessing checks for ', target_scan)
+
+        # Check if the scan has been processed completely
+        if (self.pipeline_fuse.ScanDone() & target_scan).fetch().size == 0:
```

Comment:
you can do `if len(self.pipeline_fuse.ScanDone() & target_scan) == 0:`

Reply:
Will change